plugins/foreman_discovery/2.0/index.md

```diff
@@ -0,0 +1,306 @@
+---
+layout: plugin
+title: Foreman Discovery 2.0 Manual
+version: 2.0
+---
+
+# 1. {{ page.title }}
+
+This plugin provides automatic bare-metal discovery which is implemented as a
+PXE-boot live image with minimized CentOS that is running from memory. Once
+Foreman is configured to boot the discovery image for all unknown hosts, new
+nodes calls home during the start and wait until they are provisioned.
+
+All communication is through Foreman Proxy which is running a plugin and has
+direct access both to the provisioning network and Foreman instance:
+
+    --
+    Discovered Host -> Foreman Proxy -> Foreman
+    --
+    Foreman -> Foreman Proxy -> Discovered Host
+    --
+
+## 1.1 Release Notes
+
+### 1.1.1 Foreman Discovery plugin
+
+- **2.0**: Automatic provisioning, ZIP-based extensions, support for new
+  Discovery Image 2.0, UI enhancements, new CLI plugin, reboot option and
+  two-pane layout.
+
+### 1.1.2 Foreman Proxy Discovery plugin
+
+- **1.0**: Initial version.
+
+### 1.1.3 Foreman Discovery Image
+
+- **2.0**: Smaller in size (only 150 MB whole RAM disk), much faster and
+  easier to build (dropped ovirt-node support), easier downstream building (in
+  koji), based on centos7 and fedora19, systemd and network manager full
+  support (fast and reliable boot), easier to troubleshoot.
+
+# 2. Installation
+
+There are three different components to install: Foreman Discovery plugin,
+Foreman Proxy Discovery plugin (aka Smart Proxy Discovery) and Foreman
+Discovery Image.
+
+## 2.1 Foreman Discovery plugin
+
+Installation of the plugin is standard, follow our documentation about
+[How to Install a Plugin](/manuals/latest/index.html#6.1InstallaPlugin).
+Our installer installs discovery plugin by default, manual installation can be
+done with:
+
+    # yum -y install ruby193-rubygem-foreman_discovery
+    # foreman-rake db:migrate
+    # foreman-rake db:seed
+
+Restart of Foreman is needed when doing manual installation.
+
+## 2.2 Foreman Proxy Discovery plugin
+
+To install the plugin, do:
```

Comment:
It's always required, we support direct communication for compatibility reasons. I'd rather teach users to always use it. I will leave a note about the legacy mode in the first section tho.